frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -82,6 +90,29 @@ const PolicyPage = ({
     },
   });
 
+  // // TODO(Sarah): What should happen if a user without save permissions tries to directly navigate
```

Comment:
@gillespi314 did you mean to address this, or leave it here for the future?

Reply:
Either way. If folks can chime in on the desired UX, we can handle it now. Or we can leave the todo and open a separate ticket if we want to ship a patch in the meantime. @lukeheath do you have any input on how you'd like this to be handled?